Core/Src/main.c

```diff
@@ -284,6 +413,38 @@ HAL_StatusTypeDef send_can_msg(const uint8_t *data, size_t len) {
 
     return status;
 }
+
+uint32_t Deg_To_CCR(uint8_t deg, TIM_HandleTypeDef *timer) {

```

Comment:
should all of these helper functions be in the main.h file or stay in the main file? or perhaps even a different folder with utility functions, some of which may be shareable across code bases, like reading and writing from CAN for instance. this would require combining all of the repos though, which may not be advantageous.